#!/usr/bin/env python3
 
PACKAGE = "ROS_Auto_Taxi"
 
from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 
gen.add("lb_h",    int_t,    0, "lb_h", 15,  0, 255)
gen.add("lb_s",    int_t,    0, "lb_s", 0,  0, 255)
gen.add("lb_v",    int_t,    0, "lb_v", 0,  0, 255)
gen.add("ub_h",    int_t,    0, "ub_h", 255,  0, 255)
gen.add("ub_s",    int_t,    0, "ub_s", 255,  0, 255)
gen.add("ub_v",    int_t,    0, "ub_v", 255,  0, 255)
gen.add("blurr",   int_t,    0, "blurr", 10,  0, 30)

exit(gen.generate(PACKAGE, "ROS_Auto_Taxi", "LineFollower"))
